### PR TITLE
[Doc] Add *Solr sink connector guide*

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/5015

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@tuteng could you please help review? Thank you

File: site2/docs/io-solr-sink.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Before using the Solr sink connector, you need to create a configuration file through one of the following methods.
+
+* JSON
+
+    ```json
+    {
+        "solrUrl": "localhost:2181,localhost:2182/chroot",
+        "solrMode": "SolrCloud",
+        "solrCollection": "techproducts",
+        "solrCommitWithinMs": "100",
```
Comment (tuteng) [inline site2/docs/io-solr-sink.md]: This `solrCommitWithinMs`  value seems to be an integer

```
"solrCommitWithinMs": 100,
```

**Comment by tuteng** [comment]
run java8 tests

**Comment by tuteng** [comment]
run java8 tests

**Pull request merged**